src/main/java/frc/robot/RobotContainer.java

```diff
@@ -273,7 +297,7 @@ public class RobotContainer {
   /** Singleton instance of {@link DisabledLight} for the whole robot. */
   public static DisabledLight disabledLights = new DisabledLight();
 
-  /** Singleton instance of {@link EnableLights} for the whole robot. */
+  /** Singleton instance of {@link EnabledLight} for the whole robot. */
   public static EnabledLight enabledLights = new EnabledLight();
```

Comment:
maybe change the variable name to fit the class (EnabledLights to EnabledLight)

Reply:
counterpoint, there is more than one LED.

Reply:
counterpoint you should've named the class to match that originally

Reply:
is it not named EnableLights?????

Reply:
no

Reply:
counterpoint it's an abstracted class, so of course the object name is more descriptive than the class name

also i dont want to rename a whole class

Reply:
counterpoint 2: electric boogaloo: ALL OF THE LIGHT COMMANDS ARE NAMED LIKE THAT

Reply:
```java
  /** Singleton instance of {@link DisabledLight} for the whole robot. */
  public static DisabledLight disabledLights = new DisabledLight();

  /** Singleton instance of {@link EnabledLight} for the whole robot. */
  public static EnabledLight enabledLights = new EnabledLight();

  /** Singleton instance of {@link NoteLight} for the whole robot. */
  public static NoteLight noteLight = new NoteLight();
```
this is the current state of that file, I say we rename noteLight to noteLights and be done with it because it is not that important

Reply:
im questioning my own design now

Reply:
I say just keep it as is. This is a bigger issue that should probably be fixed in its own PR. This issue wasn't even made by GKC.

Reply:
yeah you're right, this is probably out of scope

the only reason the javadoc got changed was because javadoc wasn't building